CanvasColorSpaceProposal.md

```diff
@@ -63,56 +104,72 @@ partial interface CanvasRenderingContext2D {
 * Displayed canvases must be color corrected for the display if a display color profile is available. This color correction happens downstream at the compositing stage, and has no script-visible side-effects.
-* toDataURL/toBlob produce resources tagged as being in the sRGB color space, if the encoding format supports colorspace tagging or embedded color profiles. Exception: User agents that implement the legacy behavior must not encode any color space metadata.
-* Images with no color profile, when drawn to the canvas, are assumed to already be in the canvas's color space, and require no color transformation.
-
-#### The "rec2020" color space
-* Support is optional.
-* Requires that the pixelFormat attribute be "float16".
-* This color space exists to enable wide color gamut and high dynamic range applications that use physically correct linear color operations (e.g, blending and interpolation)
-* Color space definition
-   * This color space uses the same primaries as the sRGB color space.
-   * This color space has a linear transfer function that supports arbitrary pixel values, in particular, values outside of the range [0, 1].
-* Note that gradients will be physically uniform, and thus will not match gradients in an "srgb" canvas
-* Note that pixel values written to WebGL draw buffers are now in a space with a linear transfer function, and will not necessarily match those of an "srgb" canvas.
-
-#### The pixelFormat context creation attribute
-The pixelFormat attributes specifies the numeric types to be used for storing pixel values.
-* Support for "8-8-8-8" is mandatory. All other formats ar optional.
-* When an unsupported format is requested, the format shall fall back to "8-8-8-8".
-* With formats that use integer numeric types for the color channels, the canvas pixel buffer shall store non-linear color values, encoded using the transfer curves prescribed by the specification of the current color space.
-* With formats that use floating-point numeric types for the color channels, the canvas pixel buffer shall store linear (i.e unencoded) color values.
-* Float values outside of [0,1] range can be used to represent colors outside of the chosen color gamut. This allows float pixel formats to represent all possible colors and brightness levels. How values outside of [0,1] are displayed depends on the capabilites of the device and output display. Some implementations may simply clamp these values to [0,1]. If the device and display are capable, a pixel value of (2,2,2) should be twice as bright as (1,1,1).
+* toDataURL/toBlob produce resources tagged as being in the sRGB color space, if the encoding format supports colorspace tagging or embedded color profiles.
+    * Exception: User agents that implement the legacy behavior must not encode any color space metadata.
```

Comment:
PNG is specified to use sRGB primaries/transfer by default (when no extra metadata is present).